lib/matplotlib/backends/web_backend/mpl.js

```diff
@@ -394,9 +413,10 @@ mpl.findpos = function(e) {
 mpl.figure.prototype.mouse_event = function(event, name) {
     var canvas_pos = mpl.findpos(event)
 
-    if (this.focus_on_mouseover && name === 'motion_notify')
+    if (name === 'button_press')
```

Comment:
I stand corrected, the motion_notify event is triggered regardless of focus, and the coordinates are updated (I just ran the example code).